universities/organizations.json

```diff
@@ -1,492 +1,492 @@
     "country": "OMN"
   },
   {
-    "id": "U-6859",
+    "id": "6859",
     "name": "Southampton",
     "formal_name": "University of Southampton",
     "country": "GBR"
   },
   {
-    "id": "U-6864",
+    "id": "6864",
     "name": "ITU",
     "formal_name": "IT University of Copenhagen",
     "country": "DNK"
   },
   {
-    "id": "U-6929",
+    "id": "6929",
     "name": "Warwick",
     "formal_name": "University of Warwick",
     "country": "GBR"
   },
   {
-    "id": "U-6940",
+    "id": "6940",
     "name": "Glasgow",
     "formal_name": "University of Glasgow",
     "country": "GBR"
   },
   {
-    "id": "U-7006",
+    "id": "7006",
     "name": "UCL",
     "formal_name": "University College London",
     "country": "GBR"
   },
   {
-    "id": "U-7019",
+    "id": "7019",
     "name": "HS Leiden",
     "formal_name": "Hogeschool Leiden",
     "country": "NLD"
   },
   {
-    "id": "U-7264",
+    "id": "7264",
     "name": "University of Göttin",
     "formal_name": "University of Göttingen",
     "country": "DEU"
   },
   {
-    "id": "U-7464",
+    "id": "7464",
     "name": "Brunel",
     "formal_name": "Brunel University London",
     "country": "GBR"
   },
   {
-    "id": "U-7477",
+    "id": "7477",
     "name": "Vilnius Gediminas TU",
     "formal_name": "Vilnius Gediminas Technical University",
     "country": "LTU"
   },
   {
-    "id": "U-7887",
+    "id": "7887",
     "name": "UI",
     "formal_name": "University of Iceland",
     "country": "ISL"
   },
   {
-    "id": "U-7918",
+    "id": "7918",
     "name": "St Andrews",
     "formal_name": "University of St Andrews",
     "country": "GBR"
   },
   {
-    "id": "U-7950",
+    "id": "7950",
     "name": "Birmingham",
     "formal_name": "University of Birmingham",
     "country": "GBR"
   },
   {
-    "id": "U-8002",
+    "id": "8002",
     "name": "Radboud",
     "formal_name": "Radboud University",
     "country": "NLD"
   },
   {
-    "id": "U-8102",
+    "id": "8102",
     "name": "Passau",
     "formal_name": "University of Passau",
     "country": "DEU"
   },
   {
-    "id": "U-8170",
+    "id": "8170",
     "name": "HPI",
     "formal_name": "Hasso Plattner Institute",
     "country": "DEU"
   },
   {
-    "id": "U-8311",
+    "id": "8311",
     "name": "UMONS",
     "formal_name": "Université de Mons",
     "country": "BEL"
   },
   {
-    "id": "U-8317",
+    "id": "8317",
     "name": "Bristol",
     "formal_name": "University of Bristol",
     "country": "GBR"
   },
   {
-    "id": "U-9858",
+    "id": "9858",
     "name": "UCLouvain",
     "formal_name": "Université Catholique de Louvain",
     "country": "BEL"
   },
   {
-    "id": "U-11048",
+    "id": "11048",
     "name": "TalTech",
     "formal_name": "Tallinn University of Technology",
     "country": "EST"
   },
   {
-    "id": "U-11054",
+    "id": "11054",
     "name": "TalTech",
     "formal_name": "Tallinn University of Technology",
     "country": "EST"
   },
```

Comment:
Probably unrelated, but duplicate university (`11048` and `11054`)

Reply:
One off them was wrong. After checking in ICPC only a single version exists. removed the wrong duplicate.